requirements.txt

```diff
@@ -12,4 +12,5 @@ scikit-learn==1.2.1
 scipy==1.10.0
 seaborn==0.12.2
 tensorflow==2.11.0
-tqdm==4.64.1
\ No newline at end of file
+tqdm==4.64.1
+lida==0.0.10
```

Comment:
Can you remove `lida`? It doesn't look like it's used.

Reply:
Yep. It is now removed. 

It was there because I encountered an error when first palying with the code. But it seems to work fine without it.